gcp_variant_transforms/transforms/merge_headers.py

```diff
@@ -50,35 +54,49 @@ def merge(self, first, second):
     if (not isinstance(first, vcf_header_io.VcfHeader) or
         not isinstance(first, vcf_header_io.VcfHeader)):
       raise NotImplementedError
-    self._merge_header_fields(first.infos, second.infos)
-    self._merge_header_fields(first.filters, second.filters)
-    self._merge_header_fields(first.alts, second.alts)
-    self._merge_header_fields(first.formats, second.formats)
-    self._merge_header_fields(first.contigs, second.contigs)
-
-  def _merge_header_fields(self, first, second):
-    #type: (Dict[str, OrderedDict[str, Union[str, int]]],
-    #       Dict[str, OrderedDict[str, Union[str, int]]]) -> None
+    self._merge_header_fields(first.infos, second.infos, first.conflicts,
```

Comment:
Can we have a single member conflicts dic instead of passing two dicts to every method?
The conflicts dic will be copied into first's conflicts dict at the end.